daml-lf/encoder/src/test/scala/com/digitalasset/daml/lf/archive/testing/EncodeV1Spec.scala

```diff
@@ -270,7 +270,7 @@ object EncodeV1Spec {
       case EAbs(binder, body, Some(_)) =>
         EAbs(normalizer.apply(binder), normalizer.apply(body), None)
       case ELocation(loc, expr) if loc.packageId == hashCode =>
-        ELocation(loc = loc.copy(packageId = pkgId), expr)
+        ELocation(loc = loc.copy(packageId = pkgId), normalizer.apply(expr))
```

Comment:
What does this do? (what is wrong with out it?)

Reply:
This normalizes the  body of `ELocation`. In my case the normalization of `()`, the only `ELocation` body in my example is already normalized. This would be needed if for instance I change the definition of `aValue` like:
```
 val aValue: forall (a:*). a -> a = loc(Mod, aValue, 94, 12, 94, 87) Mod:aVar;
```

For the record the normalization do 2 things: 
- It replaces the ID of the package by "self", because one cannot predict the hash of the resulting package 
- It drop the reference of abstraction, because not handle by the encoder.